src/components/MediaPlayer/VideoJSPlayer.js

```diff
@@ -68,13 +69,20 @@ function VideoJSPlayer({ isVideo, initStartTime, ...videoJSOptions }) {
         playerDispatch({ isPlaying: true, type: 'setPlayingStatus' });
       });
     }
+
+    // Clean up player instance on component unmount
+    return () => {
+      if (player) {
+        player.dispose();
+      }
+    };
   }, [player]);
 
   React.useEffect(() => {
     if (!player) {
       return;
     }
-
+    console.log('useEffect() [startTime, endTime]', startTime, endTime);
```

Comment:
And this one.

Reply:
If you don't mind, they're helping me with timing.  I'll get rid of 'em soon.